src/utils/viewer/viewer.cpp

```diff
@@ -374,6 +391,11 @@ Viewer::Viewer(const Arguments& arguments)
       .addBooleanOption("recompute-navmesh")
       .setHelp("recompute-navmesh",
                "Programmatically re-generate the scene navmesh.")
+      .addOption("camera")
+      .setHelp("camera",
+               "Specify time to load camera transformation from. Format: "
```

Comment:
This is "too complicated" for the user.
You can just provide a string as the "full path" to the file to be loaded.
e.g., user can just provide: "~/models/hahaha/cameraTransform.txt"

Reply:
Agree, and I would rename the command-line option from "camera" to something like "camera-transform-filepath".

Reply:
That is a bit too complicated, in my opinion.
Just "camera-transform-filepath" is good enough.
The save directory can just use what was implemented here.